### DISCUSSION TITLE
blog/2018/12/05/test-any-hypothesis/index

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#56] [Blog comments]
# blog/2018/12/05/test-any-hypothesis/index

Use the infer package in R to test any statistical hypothesis through simulation.

https://www.andrewheiss.com/blog/2018/12/05/test-any-hypothesis/index.html

**Comment by TrippBishopStats** [comment]
Excellent synopsis. Infer is a great package and the ModernDive text is super accessible. It's a great companion to Introduction to Modern Statistics by Cetinkaya-Rundel & Harden which I'm guessing that you're also familiar with. And they are both inexpensive texts! I like the growing trend towards teaching stats in a more fundamental way that gets away from that flow chart bollocks that you mention. I'm starting my masters in stats here in a few weeks and one of the things that I'm really excited about is getting to help teach stats and looking for better ways to train new students in the subject.

Your walk through of the infer package is, no doubt, a great way for new folks to get a taste of how well structured hypothesis testing can be. The code you type reinforces you knowledge of the methodology, which is really cool.

I'm actually curious about moving away from the NHST framework (but my knowledge of Baysian alternatives is very limited right now), but `infer` is a great way to teach folks the basics of NHST, so I'm all for it.

**Reply by andrewheiss** [reply]
If you're interested in Bayesian frameworks, check out [the incredible (and also free!) *Bayes Rules!*](https://www.bayesrulesbook.com/)

**Reply by TrippBishopStats** [reply]
That book looks great! Thanks so much for the tip!
